src/http/driver.ml

```diff
@@ -456,34 +556,52 @@ let serve_with_details
         tls_error_handler client_address exn;
         Lwt.return_unit)
   in
+  *)
 
   (* Look up the low-level address corresponding to the interface. Hopefully,
      this is a local interface. *)
-  let%lwt listen_address =
+  let listen_address =
     match network with
     | `Unix path ->
-      Lwt.return (Lwt_unix.ADDR_UNIX path)
+      `Unix path
     | `Inet port ->
-      let%lwt addresses =
-        Lwt_unix.getaddrinfo interface (string_of_int port) [] in
+      let addresses =
+        Eio.Net.getaddrinfo_stream
+          env#net interface ~service:(string_of_int port) in
       match addresses with
       | [] ->
         Printf.ksprintf failwith "Dream.%s: no interface with address %s"
           caller_function_for_error_messages interface
       | address::_ ->
-        Lwt.return Lwt_unix.(address.ai_addr)
+        address
   in
 
+  (* TODO Value of the backlog argument? *)
```

Comment:
Seems like backlog should be around 128: https://stackoverflow.com/a/36452474/20371